sys/arm64/arm64/support.S

```diff
@@ -38,32 +38,53 @@ __FBSDID("$FreeBSD$");
 
 #include "assym.inc"
 
+#if __has_feature(capabilities)
```

Comment:
```suggestion
#if __has_feature(capabilities) && !defined __CHERI_PURE_CAPABILITY__
```

Reply:
That looks correct, however there are other issues with the code so I's unlikely to work in a purecap kernel as is.